### ISSUE TITLE
Is it really necessary in this project to include both training set and test set?

### ISSUE THREAD

**Issue by zlhaa23** [GCC15/bilibili-captcha#1]
Since the CAPTCHAs are not too complicated, maybe we can achieve a good recognition rate without even training the parameters.

**Comment by LiZifan** [comment]
We never know for sure so it is better to store some training data in case
